src/heroes/bastion/sentry.opy

```diff
@@ -16,8 +16,8 @@ rule "[bastion/sentry.opy]: Initialize sentry mode":
 
     eventPlayer.disallowButton(Button.SECONDARY_FIRE)
 
-    waitUntil(eventPlayer.isOnGround(), Math.INFINITY)
-    eventPlayer.setMoveSpeed(0)
+    #waitUntil(eventPlayer.isOnGround(), Math.INFINITY)
+    #eventPlayer.setMoveSpeed(0)
```

Comment:
Wouldn't the fix be to conditionally set movespeed to 0 only when they're in sentry mode?
```suggestion
    waitUntil(eventPlayer.isOnGround(), Math.INFINITY)
    if eventPlayer.isUsingAbility1():
        eventPlayer.setMoveSpeed(0)
```